package.json

```diff
@@ -34,11 +35,13 @@
     "json-server:graphql": "node test/data/graphql/index.js",
     "lint": "eslint bin/ examples/ lib/ test/ translations/ gulpfile.js",
     "lint-fix": "eslint bin/ examples/ lib/ test/ translations/ gulpfile.js --fix",
-    "docs": "gulp docs",
+    "docs": "robo docs",
```

Comment:
What is `robo` for?

Why do you replaced gulp? And did not update dependencies?
```
yarn run v1.17.3
$ robo docs
/bin/sh: robo: command not found
```

I found out only [this package](https://www.npmjs.com/package/robo).
It was update 6 years ago, has 2-5 weekly downloads only, and has not link to github repo?

Do we really need this package?

Reply:
I search in code, and found out gulp is not used in code too.
And there are no gulp in dependencies.

What is `docs` command for?

Reply:
robo is php task runner.
It's used here for doc generation ><

Reply:
ok

I am going to cry a little bit and think what to do about it